src/client.js

```diff
@@ -334,7 +342,7 @@ forsta.messenger = forsta.messenger || {};
         /**
          * Ensure that a thread exists matching the expression argument.
          *
-         * @param {string} expression - The {@link TagExpression} for the thread's distribution.
+         * @param {string} expression - The [TagExpression]{@link external:TagExpression} for the thread's distribution.
```

Comment:
I think with TagExpression being a typedef it would replace the `string` declaration for this argument.  a la., "@param {TagExpression} expression - The thread's distribution."